src/applications/mhv-medical-records/tests/e2e/accelerated/vitals/view-vitals-list.oracle-health.update-timeframe.cypress.spec.js

```diff
@@ -46,21 +46,14 @@ describe('Medical Records View Vitals', () => {
 
     Vitals.checkUrl({ timeFrame: '2020-01' });
 
-    // This is checking that the breadcrumbs are correct
-    cy.get('va-breadcrumbs')
-      .shadow()
-      .find('li')
-      .eq(3)
+    cy.get('[data-testid="mr-breadcrumbs"]')
       .find('a')
-      .should('be.visible')
       .should('have.attr', 'href')
-      .and('include', '/vitals?timeFrame=2020-01');
+      .and('include', '/my-health/medical-records/vitals');
```

Comment:
I want to make sure we are checking for the right thing here. Are we supposed to be including `timeFrame` when linking back to the Vitals List?

Reply:
returned it back.